tools/profiling/bloat/bloat_diff.py

```diff
@@ -102,15 +104,20 @@ def _rank_diff_bytes(diff_bytes):
     text += lib + '\n\n'
     old_version = glob.glob('bloat_diff_old/%s' % lib)
     new_version = glob.glob('bloat_diff_new/%s' % lib)
+    for filename in [old_version, new_version]:
+        if filename:
+            subprocess.check_call('strip %s -o %s.stripped' %
+                                  (filename[0], filename[0]),
+                                  shell=True)
     assert len(new_version) == 1
     cmd = 'bloaty-build/bloaty -d compileunits,symbols'
     if old_version:
         assert len(old_version) == 1
-        text += subprocess.check_output('%s %s -- %s' %
-                                        (cmd, new_version[0], old_version[0]),
-                                        shell=True).decode()
-        for filename in [old_version, new_version]:
-            subprocess.check_call('strip %s' % filename[0], shell=True)
+        text += subprocess.check_output(
+            '%s --debug-file=%s --debug-file=%s %s.stripped -- %s.stripped' %
+            (cmd, new_version[0], old_version[0], new_version[0],
```

Comment:
Just FYI, no need to change anything, but now that we're using Python3 everywhere, we can use f-strings instead of % formatting, which is maybe a bit more readable. https://docs.python.org/3/reference/lexical_analysis.html#f-strings. Requires python3.6